dbt_coding_conventions.md

```diff
@@ -108,26 +151,25 @@ final as (
 
         -- use line breaks to visually separate calculations into blocks
         case
-            when my_data.cancellation_date is null and my_data.expiration_date is not null then expiration_date
-            when my_data.cancellation_date is null then my_data.start_date + 7
+            when my_data.cancellation_date is null
+                and my_data.expiration_date is not null
+                then expiration_date
+            when my_data.cancellation_date is null
+                then my_data.start_date + 7
             else my_data.cancellation_date
         end as cancellation_date,
 
-        -- use a line break before aggregations
-        sum(some_cte.field_4),
-        max(some_cte.field_5)
+        some_cte_agg.total_field_4,
+        some_cte_agg.max_field_5
 
     from my_data
-

```

Comment:
Oh, I actually like these new lines! Happy to be told otherwise though

Reply:
The proserv team has consistently never used these new lines and I personally find it to be more distracting than visually helpful